src/DynamoCore/Models/PortModel.cs

```diff
@@ -23,12 +21,12 @@ public class PortModel : ModelBase
         /// <summary>
         /// Event triggered when a port is connected.
         /// </summary>
-        public event Action<ConnectorModel> PortConnected;
+        public event Action<PortModel, ConnectorModel> PortConnected;
 
         /// <summary>
         /// Event triggered when a port is disconnected.
         /// </summary>
-        public event PortConnectedHandler PortDisconnected;
+        public event Action<PortModel> PortDisconnected;
 
```

Comment:
This badly needed cleanup.  Now the `PortDisconnected` and `PortConnected` events are much more sensible and it removes the used, non-type safe args in `PortConnectedHandler`.